http_handler_file.cc

```diff
@@ -0,0 +1,75 @@
+#include <fstream>
+#include "http_handler_file.h"
+#include "http_request.h"
+#include "http_response.h"
+
+
+namespace http {
+
+
+// Structure for mapping extensions to content types
+struct mapping {
```

Comment:
The mapping isn't a key-value store. It could be, but it'd take extra work, and the tutorial I was following it did it this way. For small amounts of keys, its about the same cost.